cluster/common.sh

```diff
@@ -485,7 +496,7 @@ function download-release-binaries() {
   local resp="y"
   if [[ ! "${KUBERNETES_SKIP_CONFIRM:-n}" =~ ^[yY]$ ]]; then
     echo "Required release artifacts appear to be missing. Do you wish to download them? [Y/n]"
-    read resp
+    read -r resp
```

Comment:
What shellcheck failure are you addressing here?

Reply:
`read` without `-r` will mangle backslashes. Technically it makes no real difference here, but it is not harmful to use `-r` and it is a better pattern. 

This one is pretty pervasive in our scripts and not fixing it will mask actual bugs when reading more complex inputs.